mods/cnc/chrome/editor.yaml

```diff
@@ -181,6 +182,16 @@ Background@SAVE_MAP_PANEL:
 			Text: Save
 			Font: Bold
 
+ScrollPanel@MAP_SAVE_VISIBILITY_PANEL:
+	Width: 220
+	Height: 44
```

Comment:
Might want to increase this to avoid the up and down buttons overlapping with the scroll bar.